lib/nrf91_lwm2m_app/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_NRF91_LWM2M_APP_FIRMWARE_UPDATE_OBJ_SUPPORT fota/settings.c)
+
+zephyr_library_sources_ifdef(CONFIG_NRF91_LWM2M_APP_LOCATION_OBJ_SUPPORT lwm2m/lwm2m_location.c)
+zephyr_library_sources_ifdef(CONFIG_NRF91_LWM2M_APP_FIRMWARE_UPDATE_OBJ_SUPPORT lwm2m/lwm2m_firmware.c)
+zephyr_library_sources_ifdef(CONFIG_NRF91_LWM2M_APP_CONN_MON_OBJ_SUPPORT lwm2m/lwm2m_connmon.c)
+zephyr_library_sources(lwm2m/lwm2m_device.c)
```

Comment:
I think we should allow to override the default implementation for the Device and Security objects as well, i. e. introduce Kconfigs at the library level as well for these objects.

Reply:
Done.